scripts/setup-minikube-for-linux.sh

```diff
@@ -2,6 +2,10 @@
 
 set -exu
 
+# Required by minikube
+sudo apt update
+sudo apt install conntrack
```

Comment:
(another option is https://github.com/akka/akka-sample-cluster-kubernetes-java/blob/master/.travis.yml#L8)